h/formatters/annotation_moderation.py

```diff
@@ -35,7 +36,9 @@ def preload(self, ids):
         return flag_counts
 
     def format(self, annotation_resource):
-        if not self._has_permission("moderate", annotation_resource.group):
+        if not self._has_permission(
+            Permission.Group.MODERATE, annotation_resource.group
```

Comment:
This is me not knowing much about H model, etc, why `Group` and not `Annotation` here?

Reply:
This is rubbish and needlessly convoluted. You're quite right that this should be annotation moderate, not group, but if you look at `h.traversal.annotation.AnnotationContext.__acl__` you'll see that `Permission.Annotation.MODERATE` is actually derived from having moderation permission on the group.

So this is very obscurely replicating the logic there.

I started to un-pick this, but it's actually not trivial to get right and I figured this was complicated enough.

Definite follow up PR though.

Reply:
If you want a real fun time look at `h.formatters.annotation_hidden.AnnotationHiddenFormatter._current_user_is_moderator()`.

It's doing a really good job of making it hard to work out, but this is actually the same check as well.